core/src/io/anuke/mindustry/world/blocks/ItemSelection.java

```diff
@@ -40,4 +40,29 @@ public static void buildItemTable(Table table, Prov<Item> holder, Cons<Item> con
 
         table.add(cont);
     }
+
+    public static void buildDrillItemTable(Table table, Array<Item> items, Prov<Item> holder, Cons<Item> consumer){
+
```

Comment:
Why there is a line break? There is no need.

Reply:
try importing the mindustry code format
~~i've done the same too many times~~

Reply:
Mindustry IntelliJ code format: https://github.com/Anuken/Mindustry/blob/master/.github/Mindustry-CodeStyle-IJ.xml